rust_icu_ecma402/src/pluralrules.rs

```diff
@@ -0,0 +1,142 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Implements the traits found in [ecma402_traits::pluralrules].
+
+use ecma402_traits;
+use rust_icu_common as common;
+use rust_icu_upluralrules as uplr;
+use std::fmt;
+
+/// Implements [ecma402_traits::pluralrules::PluralRules] using ICU as the underlying
+/// implementing library.
+pub struct PluralRules {
+    // The internal representation of rules.
+    rep: uplr::UPluralRules,
+}
+
+pub(crate) mod internal {
+    use ecma402_traits::pluralrules::options;
+    use rust_icu_sys as usys;
+
+    // Converts the trait style option type to an equivalente ICU type.
```

Comment:
"equivalente" is the Spanish spelling :)

Reply:
Done.